src/coreclr/classlibnative/float/floatsingle.cpp

```diff
@@ -40,15 +40,6 @@
 #pragma float_control(precise, off)
```

Comment:
Nit: I have noticed that there is unused `#define _copysignf (float)_copysign` a few lines above. It is left-over from times when CopySign was FCall. You delete it while you are on it.